src/agent/coverage/Cargo.toml

```diff
@@ -35,7 +35,7 @@ winapi = "0.3"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 pete = "0.9"
-procfs = "0.12"
+procfs = { version = "0.12", default-features = false, features=["flate2"] }
```

Comment:
Can we please document what we are opting out of with this specification?

Reply:
Done.